### PR TITLE
Update the README.md #1

### PR DESCRIPTION
**Pull request by Arhaan-Siddiquee**
Hey I updated the README.md file as per the #1 issue was assigned with the requirements stated. 
screencapture-markdownlivepreview-2024-10-16-02_52_20

### CODE REVIEW

**Comment by JenBauer** [comment]
Hi @Arhaan-Siddiquee, thank you for your interest in this issue.  However, @kodebae is assigned to Issue 1: #1.  Currently, the main branch does not have the projected checked in, nor have we noted the technical stack for the project.

@kodebae, can you comment here when you have the application checked in?  Then @Arhaan-Siddiquee can update this PR to match your technical stack and we can assign Issue 1 to them.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,46 @@
-# job_tracker
+# UpWiSE
 
-This is a WIP open source project to meet the following objectives:
+**UpWiSE** is an open-source project designed to help users efficiently track and organize their job applications. The app allows users to monitor their application progress, store notes, and attach relevant job links. It is created to simplify job hunting by consolidating all application details in one place.
```
Comment (JenBauer) [inline README.md:3]: This section needs to be updated to have the project name be "Job Tracker", and the group that is sponsoring this project is called UpWiSE (upwisesc.org).

File: README.md [hunk 2]
```diff
@@ -1,8 +1,46 @@
...
+
+The Job Tracker app allows users to:
+- Add and manage job applications with essential details such as job role, company, and dates.
+- Track the current status of each application (e.g., Applied, Interview, Offer).
+- Attach related notes to each job application.
+- Include an optional link for further reference (e.g., job listing URL).
+
+The aim of this project is to provide a user-friendly interface to streamline the job search process.
+
+## Technical Stack
```
Comment (JenBauer) [inline README.md:15]: Once the initial project is checked in, this section and the initial setup section need to be updated.

**Comment by kodebae** [comment]
Hi @JenBauer Kara has also shared this resource: https://supabase.com. I would still like to consider this options for this app. If @Arhaan-Siddiquee would like to take over for issue #1 researching Supabase would be the next step from here.

**Comment by JenBauer** [comment]
As there has been no response from @Arhaan-Siddiquee and Hacktoberfest closes tomorrow, I am closing this PR.

**Pull request closed without merging**